admin/setup_worker_image.sh

```diff
@@ -85,6 +85,12 @@ sudo -iu bot_compilation sdk install groovy
 wget -O julia.tgz https://julialang-s3.julialang.org/bin/linux/x64/0.6/julia-0.6.1-linux-x86_64.tar.gz
 tar xvzf julia.tgz
 
+# Swift
+wget -O swift.tar.gz https://swift.org/builds/swift-4.0.2-release/ubuntu1610/swift-4.0.2-RELEASE/swift-4.0.2-RELEASE-ubuntu16.10.tar.gz
+sudo tar -C /usr/local -xzf swift.tar.gz
+echo 'export PATH="$PATH:/usr/local/swift-4.0.2-RELEASE-ubuntu16.10/usr/bin"' | sudo -iu bot_compilation tee -a /home/bot_compilation/.profile
+sudo chmod o+r /usr/local/swift-4.0.2-RELEASE-ubuntu16.10/usr/lib/swift/CoreFoundation/module.modulemap
```

Comment:
Whoops, guess it needs to read all those files (should've figured). So this should be 
```
sudo chmod -R o+r /usr/local/swift-4.0.2-RELEASE-ubuntu16.10/usr/lib/swift/CoreFoundation/
```

Sorry for the trouble!